conductor/lib/worker.py

```diff
@@ -379,18 +384,26 @@ def kill_reporters(self):
             logger.debug('killing reporter %s', reporter)
             reporter.kill()
 
+    def kill_metricstore(self):
+        logger.debug('killing metric store %s', self.metric_store)
+        self.metric_store.kill()
+
     def stop_work(self):
         global WORKING
         WORKING = False  # stop any new jobs from being created
         self.drain_queues()  # clear out any jobs in queue
         self.kill_workers()  # kill all threads
+        self.kill_metricstore()
```

Comment:
Can you rename the method to `kill_metric_store` for consistency with the above variable `self.metric_store`?

Reply:
sure, makes sense